addons/account_voucher/views/account_voucher_views.xml

```diff
@@ -387,6 +393,7 @@
                         </page>
                         <page string="Journal Items" groups="account.group_account_user">
                             <group>
+                                <field name="account_date" groups="base.group_no_one"/>
```

Comment:
same remark as in the stable PR

Reply:
same comment as in the previous "same remark as in the stable PR" remark.
It seems you are doing some remark-ception in this task!

Reply:
:smile: 

it's you: it looks like you are doing some remark-able effort to comply with Laura's demand!

just remove the groups="base.group_no_one" and I'll be happy

Reply:
You can be happy now! :wink:

Reply:
:hugs: